Build broken with resolve errors for grolifant and okhttp-digest

The build is broken at the moment with errors like this:
```
./gradlew clean build

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'asciidoctorj'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find org.ysb33r.gradle:grolifant:0.12.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/org/ysb33r/gradle/grolifant/0.12/grolifant-0.12.pom
       - https://repo.maven.apache.org/maven2/org/ysb33r/gradle/grolifant/0.12/grolifant-0.12.pom
       - file:/Users/robertpanzer/dev/asciidoctorj/https/oss.sonatype.org/content/repositories/snapshots/org/ysb33r/gradle/grolifant/0.12/grolifant-0.12.pom
     Required by:
         project : > com.github.jruby-gradle.base:com.github.jruby-gradle.base.gradle.plugin:2.0.1 > com.github.jruby-gradle:jruby-gradle-plugin:2.0.1
         project : > com.github.jruby-gradle.base:com.github.jruby-gradle.base.gradle.plugin:2.0.1 > com.github.jruby-gradle:jruby-gradle-plugin:2.0.1 > com.github.jruby-gradle:jruby-gradle-core-plugin:2.0.1
   > Could not find com.burgstaller:okhttp-digest:1.10.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/com/burgstaller/okhttp-digest/1.10/okhttp-digest-1.10.pom
       - https://repo.maven.apache.org/maven2/com/burgstaller/okhttp-digest/1.10/okhttp-digest-1.10.pom
       - file:/Users/robertpanzer/dev/asciidoctorj/https/oss.sonatype.org/content/repositories/snapshots/com/burgstaller/okhttp-digest/1.10/okhttp-digest-1.10.pom
     Required by:
         project : > com.github.jruby-gradle.base:com.github.jruby-gradle.base.gradle.plugin:2.0.1 > com.github.jruby-gradle:jruby-gradle-plugin:2.0.1 > com.github.jruby-gradle:jruby-gradle-core-plugin:2.0.1 > io.github.http-builder-ng:http-builder-ng-okhttp:1.0.3

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
```
Happens both locally after clearing the cache, and on the CI.

@ysb33r Do you have an idea what could be the problem here?

If I upgrade to the latest alpha, 2.1.0-alpha.2 I get similar issues:
```
./gradlew clean build

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'asciidoctorj'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find org.ysb33r.gradle:grolifant60:1.0.0-alpha.4.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/org/ysb33r/gradle/grolifant60/1.0.0-alpha.4/grolifant60-1.0.0-alpha.4.pom
       - https://repo.maven.apache.org/maven2/org/ysb33r/gradle/grolifant60/1.0.0-alpha.4/grolifant60-1.0.0-alpha.4.pom
       - file:/Users/robertpanzer/dev/asciidoctorj/https/oss.sonatype.org/content/repositories/snapshots/org/ysb33r/gradle/grolifant60/1.0.0-alpha.4/grolifant60-1.0.0-alpha.4.pom
     Required by:
         project : > com.github.jruby-gradle.base:com.github.jruby-gradle.base.gradle.plugin:2.1.0-alpha.2 > com.github.jruby-gradle:jruby-gradle-plugin:2.1.0-alpha.2
         project : > com.github.jruby-gradle.base:com.github.jruby-gradle.base.gradle.plugin:2.1.0-alpha.2 > com.github.jruby-gradle:jruby-gradle-plugin:2.1.0-alpha.2 > com.github.jruby-gradle:jruby-gradle-core-plugin:2.1.0-alpha.2
   > Could not find com.burgstaller:okhttp-digest:1.10.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/com/burgstaller/okhttp-digest/1.10/okhttp-digest-1.10.pom
       - https://repo.maven.apache.org/maven2/com/burgstaller/okhttp-digest/1.10/okhttp-digest-1.10.pom
       - file:/Users/robertpanzer/dev/asciidoctorj/https/oss.sonatype.org/content/repositories/snapshots/com/burgstaller/okhttp-digest/1.10/okhttp-digest-1.10.pom
     Required by:
         project : > com.github.jruby-gradle.base:com.github.jruby-gradle.base.gradle.plugin:2.1.0-alpha.2 > com.github.jruby-gradle:jruby-gradle-plugin:2.1.0-alpha.2 > com.github.jruby-gradle:jruby-gradle-core-plugin:2.1.0-alpha.2 > io.github.http-builder-ng:http-builder-ng-okhttp:1.0.3
```

--- a/CHANGELOG.adoc
+++ b/CHANGELOG.adoc
@@ -33,6 +33,7 @@ Bug Fixes::
 
 * -s CLI option should be changed to -e to align with Asciidoctor (#1237) (@mojavelinux)
 * Column#setWidth is ignored (#1265) (@Vampire)
+* Fix unresolvable dependency of JRuby Gradle plugin. (#1275)
 
 === Compatible changes
 

--- a/build.gradle
+++ b/build.gradle
@@ -18,6 +18,11 @@ buildscript {
     }
     dependencies {
         classpath "biz.aQute.bnd:biz.aQute.bnd.gradle:5.3.0"
+        classpath("com.github.jruby-gradle:jruby-gradle-plugin:2.0.1") {
+          exclude module: 'grolifant'
+          exclude module: 'okhttp-digest'
+        }
+        classpath 'org.ysb33r.gradle:grolifant:0.17.0'
     }
 }
 
@@ -26,7 +31,8 @@ plugins {
   id "io.sdkman.vendors" version "3.0.0"
   id "signing"
   id "io.github.gradle-nexus.publish-plugin" version "1.3.0"
-  id 'com.github.jruby-gradle.base' version '2.0.1'
+  // Moved to dependency in buildscript. See https://github.com/asciidoctor/asciidoctorj/issues/1275
+  // id 'com.github.jruby-gradle.base' version '2.0.1'
   id 'codenarc'
 }
 

